suite-native/accounts/src/components/AccountListItemInteractive.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { TouchableOpacity } from 'react-native';
+
+import { AccountListItem, AccountListItemProps } from './AccountListItem';
+
+interface AccountListItemWrapperProps extends AccountListItemProps {
+    onSelectAccount?: (accountKey: string) => void;
```

Comment:
should be required for this component

Reply:
of course, thanks 562ff533f97fd7c9a2e542767323ffbdc77cf74e